### PR TITLE
single file uploading on db

### PR DESCRIPTION
**Pull request by doorooful**
- upload function implemented
- db update vaildity check
- get did from DB(Automatically generated)

### CODE REVIEW

**Comment by doorooful** [comment]
Sample postman request:
image

**Comment by doorooful** [comment]
- Testcase1: Uploading succeed
``` bash
[2023-10-05 18:28:39,133] INFO in document: class Document - upload(): Uploading document on doc table trial#1 with dname:test1.txt...
[2023-10-05 18:28:39,135] INFO in document: class Document - upload(): Fetching did(auto-generated from database) ...
```
``` result.json
{
    "result": "SUCCESS"
}
```

- Testcase2: Duplicated file name
``` bash
[2023-10-05 18:47:09,565] INFO in document: class Document - upload(): Uploading document on doc table trial#1 with dname:demo.txt...
[2023-10-05 18:47:09,565] ERROR in document: class Document - upload(): Error uploading document with name demo.txt
[2023-10-05 18:47:09,565] ERROR in document: class Document - upload(): Duplicate entry 'demo.txt' for key 'unique_dname'
[2023-10-05 18:47:09,565] INFO in document: class Document - upload(): Uploading document on doc table trial#2 with dname:demo_2.txt...
[2023-10-05 18:47:09,566] ERROR in document: class Document - upload(): Error uploading document with name demo_2.txt
[2023-10-05 18:47:09,566] ERROR in document: class Document - upload(): Duplicate entry 'demo_2.txt' for key 'unique_dname'
127.0.0.1 - - [05/Oct/2023 18:47:09] "POST /api/filter/upload HTTP/1.1" 200 -
```
``` result.json
{
    "result": "FAIL"
}
```

**Review by OSsunNO — APPROVED** [review]
Thank you so much for your effort~!

File: dt-demo-be/app_lab.py [hunk 1]
```diff
@@ -8,7 +8,23 @@
...
+def file_upload():
+    app.logger.info("API request on upload func: POST")
+    uploaded_status = "FAIL"
+    isFileExists = 'file' in request.files and request.files['file'] is not None
+    if isFileExists:
+        file = request.files['file']
+        contents = file.read().decode('utf-8')
+
+        db_conn = dbconn.connect_to_db()
+        test_doc = Document(app)
```
[the remark this answers is not in the document]
Comment (doorooful) [reply dt-demo-be/app_lab.py:25]: Variable name would be changed after implementation of `db updates on module runs`.
Please consider this name as a temporal commitment. 😀

**Pull request merged**